tests/playbooks/roles/install-csi-cinder/tasks/main.yaml

```diff
@@ -124,3 +124,15 @@
 
       cd $GOPATH/src/k8s.io/cloud-provider-openstack
       go test -v ./cmd/tests/cinder-csi-e2e-suite/cinder_csi_e2e_suite_test.go -ginkgo.v -ginkgo.progress -ginkgo.skip="\[Disruptive\]" -ginkgo.focus="\[cinder-csi-e2e\]" -ginkgo.noColor -timeout=0 -report-dir="/var/log/" | tee "/var/log/cinder-csi-e2e.log"
+
+- name: Collect pod logs for debug purpose
```

Comment:
we can check if above command e2e tests reports failures, then only collect the logs, else we can ignore ?

Reply:
sometimes the log is also helpful for success case (e.g check functions really called etc) 
doesn't cost too much to collect those logs?

Reply:
sounds good

Reply:
@jichenjc I have one doubt regarding this, when the above task (Run functional tests for csi-cinder-plugin) fails, will this task gets executed ?